internal/cli/cli.go

```diff
@@ -49,6 +49,14 @@ func Run(ctx context.Context, io *iostreams.IOStreams, args ...string) int {
 		return 0
 	default:
 		printError(io.ErrOut, cs, cmd, err)
+
+		var _, _, e = cmd.Find(args)
+
```

Comment:
this is a nitpick, take it or leave it. 

the extra line usually reads bad. The common Go style for error checking is for it to happen right after the function call. Also using `_, e := cmd.Find(args)` is preferred to `var _, e = cmd.Find(args)`  

```go
 _, _, e := cmd.Find(args)
if e != nil {
  // handle error
}
```

Reply:
I'll take it 😄 

https://github.com/superfly/flyctl/pull/2112/commits/01b3e8eba8398f821ed62070c77257775c2b258e